dandi/metadata.py

```diff
@@ -613,6 +617,7 @@ def generate_context():
     fields = {
         "@version": 1.1,
         "dandi": "http://schema.dandiarchive.org/",
+        "dandiasset": "http://iri.dandiarchive.org/",
```

Comment:
is `-` allowed? then I would have preferred
```suggestion
        "dandi-asset": "http://iri.dandiarchive.org/",
```

Reply:
not a big fan of `-` in prefixes, but i won't complain if you really insist.

Reply:
I do not have too strong of a preference to insist. oh well, since we have `dandiset`, I guess `dandiasset` kinda follows .  I will let it be